addons-web-sdk/samples/hello-world/src/index.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { meet } from '@googleworkspace/meet-addons/meet.addons';
+
+// TODO: Make sure that you modify these constants, if you fork this!
+const CLOUD_PROJECT_NUMBER = "12345678910";
```

Comment:
Does this qualify as a secret? I wouldn't think so but want to check.

Reply:
I think the answer must be no, right, because otherwise our SDK is all compromised, since you can grab the CPN from any screenshare-promoted add-on by dropping a breakpoint in the JS. e.g. in Miro:

<img width="2040" alt="Screenshot 2024-08-20 at 10 49 34 AM" src="https://github.com/user-attachments/assets/a146ef8d-7066-4d11-aebf-4f04e911fbad">

Reply:
Oh yeah, that tracks. Sorry, flubbed that a bit. If it needs to be sent to the client JS it can't be a secret really; attackers could just grab it out of the console.